app/(auth)/register/page.tsx

```diff
@@ -0,0 +1,207 @@
+'use client'
+import Image from 'next/image'
+import Link from 'next/link'
+import { FormEvent, useState } from 'react'
+
+import Logo from '~/assets/images/logo.png'
+import CheckboxInput from '~/components/ui/form/checkbox'
+import Label from '~/components/ui/form/label'
+import SelectInput from '~/components/ui/form/select'
+import { DataSelect } from '~/components/ui/form/select/types'
+import TextInput from '~/components/ui/form/text-input'
+import TextAreaInput from '~/components/ui/form/textarea'
+import countries from '~/data/countries.json'
+import clsxm from '~/utils/clsxm'
+
+const RegisterPage = () => {
+  const [isSubmit, setIsSubmit] = useState(false)
+  const [country, setCountry] = useState<DataSelect | undefined>()
+  // const [country, setCountry] = useState<string>()
+
+  const onSubmit = (e: FormEvent) => {
+    e.preventDefault()
+    setIsSubmit(true)
+  }
+
+  return (
+    <div className="flex min-h-screen w-full flex-col items-center justify-center bg-custom-white-broken">
+      <div className="mt-8 mb-9 flex items-center justify-center overflow-hidden rounded-full bg-white p-3 shadow-md">
+        <Image src={Logo} alt="logo site" width={75} height={75} />
+      </div>
+
+      <div className="container flex flex-row justify-center">
+        <form
+          className={clsxm(
+            'bg-w group h-min basis-1/2 rounded border-t-2 border-t-custom-purple bg-white shadow-sm',
```

Comment:
`bg-w`? `h-min`?
Did you try using some plugins or what?

Reply:
Okay I will check it